api/src/routes/guilds/#guild_id/members/#member_id/index.ts

```diff
@@ -25,13 +25,19 @@ router.patch("/", route({ body: "MemberChangeSchema" }), async (req: Request, re
 
 	const member = await Member.findOneOrFail({ where: { id: member_id, guild_id }, relations: ["roles", "user"] });
 	const permission = await getPermission(req.user_id, guild_id);
+	const everyone = await Role.findOneOrFail({ guild_id: guild_id, name: "@everyone", position: 0 });
```

Comment:
convert this into a pseudo role (i.e. synthesised from member list and default channel overrides) in the next refactor